services/motion/builtin/builtin.go

```diff
@@ -296,6 +297,21 @@ func (ms *builtIn) planMoveOnGlobe(
 	limits := []referenceframe.Limit{
 		{Min: -straightlineDistance * 3, Max: straightlineDistance * 3},
 		{Min: -straightlineDistance * 3, Max: straightlineDistance * 3},
+		{Min: -2 * math.Pi, Max: 2 * math.Pi},
+	}
+
+	if extra != nil {
+		if profile, ok := extra["motion_profile"]; ok {
+			motionProfile, ok := profile.(string)
+			if !ok {
+				return nil, nil, errors.New("could not interpret motion_profile field as string")
+			}
+			if motionProfile == motionplan.PositionOnlyMotionProfile {
```

Comment:
This can be written `kinematicsOptions.PositionOnlyMode = motionProfile == motionplan.PositionOnlyMotionProfile`